### PR TITLE
Data-Driven Decisions Blog Post, writing and code deliverables

### PR DESCRIPTION
**Pull request by kylebegovich**
### Reference to issue
- #29 and #94

### Description of the changes proposed in the pull request
- New Jupyter notebook
- Parallel HackMD file available here: https://hackmd.io/@9p8S2u5RRPWZbntOMX167w/SkYSqUPsU

### Reviewers requested:
- @lazyplatypus 
- @AlexDoytch

### CODE REVIEW

File: DDD Workshop.ipynb [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+    "\n",
+    "These are the things that you want in goal setting for sure, but we also want them in information, and it's no coincidence (p > 0.05). We want information that supports the goals you have when using it. Otherwise, information is as good as trivia, which is not bad necessarily, but useless by definition. Very briefly point by point:\n",
+    "\n",
+    "- Specific: this information should be scoped to answer the question(s) you have, as information too broad risks being less applicable to the situation\n",
+    "- Measurable: an excellent metric for what to do with your data, and the subject of the next part\n",
+    "- Attainable: similar to specificity, information too specific might be too difficult to build a solution for, or not worth the development time\n",
+    "- Relevant: if your information is not about your subject, it won't be a good basis for solutions\n",
+    "- Timely: while data being too old doesn't mean that it's wrong, it makes the correlation to today weaker and harder to build for. Likewise, a projection too far in the future might not be appropriate for your current project or plans.\n",
```
Comment (AlexDoytch) [inline DDD Workshop.ipynb:80]: I feel like you could merge these into one set of bullet points

File: DDD Workshop.ipynb [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+    "\n",
+    "#### What makes a high-level conclusion?\n",
+    "\n",
+    "These types of conclusions are for very broad generalizations and are mostly observations, rather than predictions. There is a way to categorize the types of insights gained from data analysis: Descriptive, Diagnostic, Predictive, and Prescriptive. These can be thought of as increasing both specificities and work required: \n",
+    "\n",
+    "![](https://i.imgur.com/HlYuOwH.png)\n",
+    "\n",
+    "<!-- cite: https://www.scnsoft.com/blog/4-types-of-data-analytics -->\n",
+    "\n",
+    "A high-level conclusion is then one of the first two of these and is broader in scope. For example, in looking at the flight data we see that there are twice as many flights to LA relative to Vancouver or New York. This information is probably easy to see, and we can confirm it over the entire dataset:"
```
Comment (AlexDoytch) [inline DDD Workshop.ipynb:176]: Meaning descriptive and diagnostic? A little unclear

File: DDD Workshop.ipynb [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+    "\n",
+    "So what do we do with this? Well, there is a tally of people living in the U.S. since its founding, and that data is aggregated and made available to the public. The individual entries are removed to protect people's privacy, but the country is big enough that the aggregates can often be quite granular. A nice repository of information is [available online](https://www.census.gov/population/www/censusdata/hiscendata.html) and is used every day for things like redistricting, allocating federal funds, and business decisions. The last of these is probably most applicable to us here today, as the types of business decisions feed us into the types of conclusions we want to draw from this information, turning knowledge into conclusions.\n",
+    "\n",
+    "\n",
+    "#### Flight data\n",
```
Comment (AlexDoytch) [inline DDD Workshop.ipynb:130]: Where would the reader access flight data? I couldn't find a link to the repository in the article

File: DDD Workshop.ipynb [hunk 4]
```diff
@@ -0,0 +1,398 @@
...
+    "\n",
+    "for line in data_file:\n",
+    "    airport = line[:3]\n",
+    "    if airport == \"YVR\":\n",
+    "        fields = line.split(\",\")\n",
+    "        num_seats = int(fields[1])\n",
+    "        occ_percent = float(fields[2])\n",
+    "        occ_count = floor(num_seats * occ_percent)\n",
+    "        line = \"{},{},{},{}\".format(fields[0], fields[1], occ_count, fields[3])\n",
+    "        \n",
```
Comment (AlexDoytch) [inline DDD Workshop.ipynb:234]: Clarifying the programming language and adding 2-3 sentences about the code you ran would help, as this is an introduction post, so many people will not have much experience. For example, here you can describe the difference between regular and cleaned flight data, then explain what the declaration of the line variable is doing.

File: DDD Workshop.ipynb [hunk 5]
```diff
@@ -0,0 +1,398 @@
...
+    "\n",
+    "Ad agencies would be jumping from their seats to start hiring male actors between 18 and 25! This touches on how useful information can be, and, as a bonus, is generally a good way to tell how good the question being asked in a survey was.\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Datasets\n",
+    "\n",
+    "On our journey to data-enlightenment, we do need to start somewhere. I've enjoyed sharing comics and other cherry-picked factoids for you all, but let's now practice what we preach.\n",
+    "\n",
+    "#### Covid-19 data\n",
```
Comment (AlexDoytch) [inline DDD Workshop.ipynb:101]: Part one does a great job of capturing the scope of data-driven decisions, but I think we could take one of the COVID-19 or census data sections and remove them.

File: DDD Workshop.ipynb [hunk 6]
```diff
@@ -0,0 +1,398 @@
...
+    "\n",
+    "\n",
+    "\n",
+    "# Part 3: Takeaways\n",
+    "\n",
+    "## Data Visualizations\n",
+    "\n",
+    "There is a huge elephant in the room here: sometimes data needs to be communicated from the analyst to other people. The analyst might be the one making decisions and might not be, and the decision-maker might be data conscious and might not be, so how do we bridge that gap? An extensive blog post about making data-driven decisions? NO! Data visualizations of course!\n",
+    "\n",
```
Comment (AlexDoytch) [inline DDD Workshop.ipynb:349]: Is it possible to add a brief data visualization example using flight data here? For example, take the output text data from the previous example and convert it to a bar graph grouped by airport.

File: DDD Workshop.ipynb (hunk 3 above)
Comment (kylebegovich) [reply DDD Workshop.ipynb:130]: This is a great question, in my Jupyter notebook I have it as a linked file, but I'm not sure how blog posts get published on the site if there's a way to attach the text as well. This might be a question for the eng team.

File: DDD Workshop.ipynb (hunk 5 above)
Comment (kylebegovich) [reply DDD Workshop.ipynb:101]: Good call, I'm going to comment out the census data, the observation I had about the counts can be part of a post that discusses that, if it's something we develop in the future

File: DDD Workshop.ipynb (hunk 2 above)
Comment (kylebegovich) [reply DDD Workshop.ipynb:176]: good point, updated to backward/forward-looking

**Comment by kylebegovich** [comment]
_should_ have addressed all the suggestions, please take another quick look @AlexDoytch

**Comment by AlexDoytch** [comment]
Content wise it looks good but for some reason only part 1 is not displaying correctly :(

image

**Comment by kylebegovich** [comment]
aight should be fixed now

File: DDD Workshop.ipynb [hunk 7]
```diff
@@ -0,0 +1,547 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data-Driven Decisions\n",
+    "\n",
+    "In this piece, we'll get down to the core of what data means, how we can extract knowledge from information from data, and how to use it as a savvy businessperson, researcher, or analyst. You won't need any preparation, but there will be some code available for the aspiring programmers to use and modify. You will walk away from this page with a better understanding of what your data needs or needs not and how to most effectively to use it.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:9]: "Needs not" sounds a little bit off to me, maybe going with something simpler like "doesn't need" would work better.

File: DDD Workshop.ipynb [hunk 8]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "\n",
+    "\n",
+    "# Part 1: Information vs Data\n",
+    "\n",
+    "Webster's online dictionary defines Data as factual measurements or statistics used as a basis for actions such as reasoning, deliberations, or calculation. We want to pay specific attention to the word \"basis\" here, as it highlights the fact that data can be, quite easily, manipulated. If we don't want to be susceptible to this manipulation, then we should make sure we turn that data into information. We see that Information's entry is \"knowledge obtained from investigation, study, or instruction,\" and that is really what's at play when we're making smart, data-driven decisions:\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:15]: - Instead of Data, use "data" 
- Remove the , between "here" and "as" 
- Would remove the commas around "quite easily" to emphasize that it's easy to manipulate data
- Is "Information's entry" referring to the definition in the Webster's online dictionary? Could clarify if so; maybe something like this: We also see that "information" is the same diction is knowledge obtained from..."

File: DDD Workshop.ipynb [hunk 9]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "## Fallacies and Manipulation\n",
+    "\n",
+    "<!-- cite: https://www.datasciencecentral.com/profiles/blogs/data-fallacies-to-avoid-an-illustrated-collection-of-mistakes -->\n",
+    "\n",
+    "The first step in being a conscious member of the data community is to identify and refute bad data appropriately. There are many resources for the many, varied ways to manipulate data, but we'll look at only the most common and most relevant ones here.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:21]: - Reword second sentence to "there are many resources for the various ways data is manipulated..."

File: DDD Workshop.ipynb [hunk 10]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "#### Confirmation Bias\n",
+    "\n",
+    "The biggest consequence (good or bad) of the information age, is that data is available at the click of a mouse or wake-word in almost everybody's homes. It certainly is good for us to have access to more, but that means that we now have to be aware of confirmation bias as we interpret this data. If you're looking for the answer to a question that you don't have an opinion on, even slight differences in phrasing will affect the content of your results. Take Coffee and Acne for example, in a simple Google search question we see very different results based on the choice of cause/cure:\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:25]: - Remove comma between "age" and "is"
- Would reword to something like: For example, look at how we see very different results conduct a seemingly simple Google search questions about whether coffee cures or causes acne

File: DDD Workshop.ipynb [hunk 11]
```diff
@@ -0,0 +1,547 @@
...
+    " \n",
+    " Cure            |  Cause\n",
+    ":-------------------------:|:-------------------------:\n",
+    "![](https://i.imgur.com/z1FkMNB.png)  |  ![](https://i.imgur.com/qIPZvc2.png)\n",
+    "\n",
+    "We see that even in a case where you're not trying to put your bias into the search, that your results can be skewed. This can be resolved by looking at aggregates of data instead of single-sources. This makes way for \"pop science\" where a study finds that X causes/cures Y, where both X and Y are common conditions. We see more how these unfortunate statistical anomalies become daytime news stories here.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:31]: - Would reword to condense: Even if you're trying to put bias into your search, your results can be skewed.

File: DDD Workshop.ipynb [hunk 12]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "\n",
+    "#### Data Dredging\n",
+    "\n",
+    "Data dredging is the effect when intentional or not, you have too many variables or opportunities for correlation for a single dataset. There is a famous example of this, when the website FiveThirtyEight reported on the specific field of nutrition, as the data available has issues already. They noted the specific issues of how they generate \"huge data sets with many, many variables,\" which are the most susceptible data dredging. [The article](https://fivethirtyeight.com/features/you-cant-trust-what-you-read-about-nutrition/) finds a few particularly remarkable relationships, where they found the following links from the respondent data:\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:36]: - Would reword to: Data dredging is the effect when you have too many variables or opportunities (whether intentional or not) for correlation for a single dataset. 
- Would condense to: A famous example of this is when the website FiveThirtyEight reported on the specific field of nutrition, as the data available has issues already.

File: DDD Workshop.ipynb [hunk 13]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "\n",
+    "#### P-Hacking and Cherry Picking\n",
+    "\n",
+    "You might have noticed the far-right column in the above picture has a \"p-value\" label. This is the value that statisticians use to measure the risk of a false positive. In general, studies with a p-value of less than 0.05, or 5%, can be published. It is a hard burden to prove in some situations, but in some situations it is absurdly easy. Linked in the above article is an [interactive piece](https://projects.fivethirtyeight.com/p-hacking/) by FiveThirtyEight that allows you to use p-hacking in action. This one comes with a bonus of letting you lean into confirmation bias as well, as it contains economic and political data from the last ~70 years. You get to cherry-pick the types of political data to use and the ways to measure a \"good\" economy, and the right panel allows you to see how significant your results are. It should go without saying that if you cherry-pick enough, you can prove any conclusion you want to.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:45]: - Would say: from the last 70 years.

File: DDD Workshop.ipynb [hunk 14]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "This phenomenon is very possible in the transfer of information at any stage from data sources to scientists to reporters to consumers, again intentional or unintentional. We see this taken to a comedic level with XKCD's exploration of jelly beans:\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:47]: Would reword to: This phenomenon is very possible during the transfer of information at any stage — whether from  from data sources to scientists to reporters to consumers — and it can be intentional or unintentional.

File: DDD Workshop.ipynb [hunk 15]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "What you should take away here is that data manipulated is not meaningful, and I argue it is not even information, as it does not represent knowledge in any real way. Let's not be too cynical now, there _is_ meaningful information in the world, and we want to separate it from this darker part of the data sphere.\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Meaningful Information\n",
+    "\n",
+    "Here we have information, not misleading or malicious, and want to determine what to do with it. So then, what makes your information worth producing and using?\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:60]: Reword to: ... goal-setting, and it stands for:

File: DDD Workshop.ipynb [hunk 16]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "#### \"Useful\" Information\n",
+    "\n",
+    "When we say \"useful\", what we want is SMART. SMART is an acronym that is generally used for goal-setting, where it stands for:\n",
+    "\n",
+    "- Specific: this information should be scoped to answer the question(s) you have, as information too broad risks being less applicable to the situation\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:66]: as information that is too broad risks being less applicable to the situation

File: DDD Workshop.ipynb [hunk 17]
```diff
@@ -0,0 +1,547 @@
...
+    "- Measurable: an excellent metric for what to do with your data, and the subject of the next part\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:67]: - Capitalize "an" at the beginning 
- The subject of the next part of what? Not sure if this segment is unfinished here.

File: DDD Workshop.ipynb [hunk 18]
```diff
@@ -0,0 +1,547 @@
...
+    "- Attainable: similar to specificity, information too specific might be too difficult to build a solution for, or not worth the development time\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:68]: -Capitalize "similar"
- Add . after time

File: DDD Workshop.ipynb [hunk 19]
```diff
@@ -0,0 +1,547 @@
...
+    "- Relevant: if your information is not about your subject, it won't be a good basis for solutions\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:69]: Capitalize beginning of what follows the colon and add period at the end

File: DDD Workshop.ipynb [hunk 20]
```diff
@@ -0,0 +1,547 @@
...
+    "- Timely: while data being too old doesn't mean that it's wrong, it makes the correlation to today weaker and harder to build for. Likewise, a projection too far in the future might not be appropriate for your current project or plans.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:70]: Capitalize While

File: DDD Workshop.ipynb [hunk 21]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "\n",
+    "<!-- cite https://www.mindtools.com/pages/article/smart-goals.htm -->\n",
+    "\n",
+    "These are the things that you want in goal setting for sure, but we also want them in information, and it's no coincidence (p > 0.05). We want information that supports the goals you have when using it. Otherwise, information is as good as trivia, which is not bad necessarily, but useless by definition.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:75]: Would pick a word that is more specific than "things." Maybe reword to something like this: All of the above are important to keep in mind when goal setting. We also want them in information, and it's no coincidence...

File: DDD Workshop.ipynb [hunk 22]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "\n",
+    "#### \"Strong\" Information\n",
+    "\n",
+    "While this sounds like a dataset for powerlifting competitions, it reflects the next stage in data-driven decision making that we need to address, what do we do with the data?\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:80]: that we need to address: what do we 
* important to switch comma to colon

File: DDD Workshop.ipynb [hunk 23]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "\n",
+    "\n",
+    "## Datasets\n",
+    "\n",
+    "On our journey to data-enlightenment, we do need to start somewhere. I've enjoyed sharing comics and other cherry-picked factoids for you all, but let's now practice what we preach.\n",
+    "\n",
+    "#### Covid-19 data\n",
+    "\n",
+    "The ongoing pandemic is something that has had lots of impact on the world, as I'm sure you have already noticed. The ways that it has had an impact is measurable, both to the human body and to our human society, and many high-stakes decisions depend on understanding this data. The sources online are variant, with online repositories like [Kaggle](https://www.kaggle.com/datasets) and [Google Research](https://datasetsearch.research.google.com/) and official sources like [data.gov](http://www.data.gov) and the [CDC](https://www.cdc.gov/datastatistics/index.html).\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:98]: Would reword to: There is an abundance of sources online, such as online repositories like

File: DDD Workshop.ipynb [hunk 24]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "There are many data visualizations and interactives as well as experts scrutinizing and summarizing this data for us all, thankfully. If you are looking for a kit project or want to practice with some visualization tools, however, this might be a good place to start working.\n",
+    "\n",
+    "One of these that is particularly well-made is the [arcgis dashboard](https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6), which aggregates data and allows you to see the impact on your local county as well as the national/international trends. It does this without leaving room for p-hacking by allowing inappropriate filtering or manipulation, but still answering the questions it seeks to address. You can see the trendline for every state, as well as any country by selecting it in the list on the left. See here:\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:102]: Condense to "A particularly well-made example is..."

File: DDD Workshop.ipynb [hunk 25]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "<!-- Cite: https://www.census.gov/history/pdf/Article_1_Section_2.pdf -->\n",
+    "\n",
+    "<!-- I don't want to make historical judgments on the moral quality of this data, but the statistical quality of the data is prevalent here. We see that there is 10-year periodic data-gathering, a full count of \"free Persons\", and a multiple on \"all other Persons.\" This means that the ratification of the 14th amendment (that rescinded the three-fifths compromise) in 1868 should be measurable in the differences in the 1860 and 1870 censuses.\n",
+    "\n",
+    "I DID NOT SEE THE CORRELATION I SUPPOSED HERE... I WILL INVESTIGATE FURTHER, BUT THIS SEEMS LIKE AN INTERESTING FOLLOW UP ARTICLE IF THE MYSTERY IS RESOLVED IN AN INTERESTING WAY-->\n",
+    "\n",
+    "What all of that text means is that every 10 years, falling on the years ending in 0, there is a nationwide tally of the people who live in the country. There is some historical context here of course as well, including the exception to Indian reservations that are not _really_ part of the U.S. in lots of ways, and the three-fifths compromise that was removed in the 14th amendment.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:120]: Looks like the other parts of the Census Data section were redacted, so i'm not sure if for this and the next section talking about the census you wanted to move the heading down here too?

File: DDD Workshop.ipynb [hunk 26]
```diff
@@ -0,0 +1,547 @@
...
+    "&vellip;\n",
+    "\n",
+    "We will access the full 1800 line file in the code, but all of the contents will follow the format above.\n",
+    "\n",
+    "\n",
+    "## Types of Conclusions\n",
+    "\n",
+    "#### What makes a backward-looking conclusion?\n",
+    "\n",
+    "These types of conclusions are for very broad generalizations and are mostly observations, rather than predictions. There is a way to categorize the types of insights gained from data analysis: Descriptive, Diagnostic, Predictive, and Prescriptive. These can be thought of as increasing both specificities and work required: \n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:165]: Lower case the categories of data analysis to descriptive, prescriptive,...

File: DDD Workshop.ipynb [hunk 27]
```diff
@@ -0,0 +1,547 @@
...
+    "data_file.close()\n",
+    "\n",
+    "print(counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Other similar backward-looking conclusions might be is the size of the planes is related to the destinations, is the occupancy is related to the plane sizes alone, etc. Most backward-looking conclusions can be drawn from the data directly, provided you are aware of the potential fallacies and have verified your data.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:209]: Reword to: Other similar backward-looking conclusions might include the size of the planes related to the destinations, the occupancy related to the plane sizes alone, etc. Most backward-looking conclusions can be drawn from the data directly, provided you are aware of the potential fallacies and have verified your data.

File: DDD Workshop.ipynb [hunk 28]
```diff
@@ -0,0 +1,547 @@
...
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see that any backward-looking conclusion is an observation more than it is analysis. It necessarily does not try to  make any claims about new data coming in or hypotheticals, those would be in the next section.\n",
+    "\n",
+    "\n",
+    "#### What makes a forward-looking conclusion?\n",
+    "\n",
+    "While it would feel natural to draw a dichotomy here, it would be inappropriate to do so. Forward-looking or specific conclusions depend highly on the backward-looking conclusions to make prescriptive or prospective claims. The difference between these two is that prescriptive advocates for a specific next action, the way we might want for making good, data-driven decisions, while prospective is claiming what is likely to happen. Understand that the difficulty with prescription is causality, that doing one thing would cause another, when prescriptive simply acknowledges a connection of some kind. We go back to XKCD to explore another application of bad statistical thinking:\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:249]: Rewrite to: Understand that the difficulty with prescription is causality (doing one thing would cause another) while prescriptive simply acknowledges a connection of some kind.

File: DDD Workshop.ipynb [hunk 29]
```diff
@@ -0,0 +1,547 @@
...
+    "    rate_by_airport_time[key] = occupancy_average\n",
+    "\n",
+    "[print(k, round(v, 4)) for k,v in rate_by_airport_time.items()]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There we go! We can now make a conclusion about which flights would be best to add to the current schedule. From the looks of it, another redeye to JFK would be the best bet here, but this relies on the assumption that the additional supply would produce additional demand. If the demand is static for redeye flights, meaning the supply changing has little effect on the demand, then adding flight here would do very little. It is also possible that the current flights, even though they are the most full, do not have any more demand for additional seats. These aren't questions we can answer with this dataset alone but might be one you could get from somebody else working at the airport.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:334]: Reword last sentence here to: These aren't questions we can answer with this dataset alone, but maybe you could get some answers from somebody else working at the airport.

File: DDD Workshop.ipynb [hunk 30]
```diff
@@ -0,0 +1,547 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see the issue with the original data set very clearly here, and we see that the conclusion from part 2 is still correct (JFK Redeye has the highest percentage), but we also see how small the percentage difference is.\n",
+    "\n",
+    "\n",
+    "There are many ways to communicate the conclusions that you have gathered, and there are many ways that those conclusions can be presented. We want to make sure that in our effort to convince, we're not stepping over the line of manipulating the other person. There are some heinous examples of misleading visualizations, some of the worst posted online and easy to search up in [aggreator lists](https://analythical.com/blog/examples-of-awful-data-visualization). It is worth taking some time to see the ways these can be misleading too, as the common mistakes can creep into your visualizations too if you're not careful. I've found [this guide from venngage](https://venngage.com/blog/misleading-graphs/) quite useful in familiarizing myself with the do not's of data viz.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:496]: Condense: It is worth taking time to see the ways these can be misleading too, as the common mistakes can creep into your visualizations if you're not careful.

File: DDD Workshop.ipynb [hunk 31]
```diff
@@ -0,0 +1,547 @@
...
+    "\n",
+    "And finally, the complete guide, a bit old and extensive but still incredibly relevant as a resource: [The Quartz guide to bad data](https://qz.com/572338/the-quartz-guide-to-bad-data/). This piece is **HUGE**, but if you're looking for something and can't find it anywhere else, it likely lives in the tomes of Quartz.\n",
+    "\n",
+    "\n",
+    "## Resource and Reference Links\n",
+    "\n",
+    "I've aggregated the links to most resources throughout the article in order here, for reference, or for further exploration, which I would highly encourage for any aspiring data scientists or teachers among you.\n",
```
Comment (allenaavila) [inline DDD Workshop.ipynb:503]: I've aggregated the links to most resources throughout the article in order here for reference or for further exploration, which I would highly encourage for any aspiring data scientists or teachers among you.
